build.sbt

```diff
@@ -84,19 +84,28 @@ lazy val compilationSettings = Seq(
     , "-feature"
     , "-language:_"
     , "-Ywarn-value-discard"
-    , "-Yno-adapted-args"
     , "-Xlint"
     , "-Xfatal-warnings"
     ) ++ (
       CrossVersion.partialVersion(scalaVersion.value) match {
         case Some((2, 10)) =>
+          Seq("-Yno-adapted-args")
+        case Some((2, 13)) =>
           Seq.empty
         case _ =>
-          Seq("-Ywarn-unused-import")
+          Seq("-Yno-adapted-args", "-Ywarn-unused-import")
       }
     )
   , scalacOptions in (Compile,console) := Seq("-language:_", "-feature")
   , scalacOptions in (Test,console) := Seq("-language:_", "-feature")
+  , unmanagedSourceDirectories in Compile ++= {
+      (unmanagedSourceDirectories in Compile).value.map { dir =>
+        CrossVersion.partialVersion(scalaVersion.value) match {
+          case Some((2, 13)) => file(dir.getPath ++ "-2.13")
+          case _             => file(dir.getPath ++ "-2.10-2.12")
```

Comment:
I'm very happy to go with whatever is the default convention across scala projects, but I noticed ScalaCheck used `2-13.+` and `2.13-` which I quite like for brevity. I'll defer to whatever you think is best here.

https://github.com/rickynils/scalacheck/tree/master/src/main

Reply:
:+1: It seems that way is suggested in https://github.com/scala/collection-strawman/wiki/FAQ#how-do-i-cross-build-my-project-against-scala-212-and-scala-213